packages/datetime/src/common/dateUtils.ts

```diff
@@ -14,14 +14,22 @@
  * limitations under the License.
  */
 
-import { DateRange } from "./dateRange";
+import { isSameDay } from "date-fns";
```

Comment:
pulled in these implementations from datetime2 package, updating some names to match "date-fns"